src/routes/costModels/costModelsDetails/utils/table.tsx

```diff
@@ -40,6 +42,12 @@ export function getRowsByStateName(stateName: string, data: any) {
     ];
   }
   return data.map((item: CostModel) => {
+    const dateTime = intl.formatMessage(messages.dateTime, {
+      dateTime: formatDate(
+        new Date(new Date(item.updated_timestamp).toISOString().replace(/z$/i, '')),
```

Comment:
new Date will convert the time to local because of the Z at the end and would then give a "-5 GMT" for example... as we are adding UTC timestamp ourselves, I'm removing the Z to prevent this

Reply:
that's actually a separate dependency, is that okay?

Reply:
I think using date-fns-tz would be ok. When I created the issue, it was because we were using two completely unrelated packages for dates. 

The date-fns-tz package is an extension of date-fns, but probably not a full replacement. We use a few more functions that date-fns-tz does not appear to support; endOfMonth, getDate, getYear, startOfMonth, etc.

Reply:
I would be ok using date-fns-tz along with date-fns. When I created the issue, it was because we were using two unrelated packages for dates. The date-fns-tz package is an extension of date-fns, so seems like a better fit.

The date-fns-tz package probably is not a full replacement for date-fns. We're using a few more functions, which date-fns-tz does not appear to support; endOfMonth, getDate, getMonth, getYear, startOfMonth, etc.